lib/presentation/screens/greeting_screen/greeting_layout.dart

```diff
@@ -7,7 +7,9 @@ import 'package:water_tracker/presentation/widgets/molecules/custom_button.dart'
 import 'package:water_tracker/presentation/widgets/molecules/greeting_widget.dart';
 
 class GreetingLayout extends StatelessWidget {
-  const GreetingLayout({super.key});
+  const GreetingLayout({super.key, required this.email});
+
+  final String email;
```

Comment:
could we instead of passing email all over the screens just use inherited widget above?

Reply:
or save it to the repository and store it there? i think, this is preferable